src/layout/SimpleTable/config.ts

```diff
@@ -0,0 +1,52 @@
+import { CG } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Presentation,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+    renderInTabs: true,
+  },
+  functionality: {
+    customExpressions: false,
+  },
+})
+  .extends(CG.common('LabeledComponentProps'))
+  .addProperty(new CG.prop('title', new CG.str()))
+  .addProperty(
+    new CG.prop(
+      'columns',
+      new CG.arr(
+        new CG.obj(
+          new CG.prop('id', new CG.str()),
+          new CG.prop('title', new CG.str()),
+          new CG.prop(
+            'component',
```

Comment:
I would probably call this `content` instead of `component`. The latter has a specific meaning for app developers, and we use it in `Grid` to point to other components in the layout. Here, as it's a mini-layout with types `Text` and `Link`. Since we already have an actual layout-component called `Text`, this might look confusing to app developers.